docs/xshareclasses.md

```diff
@@ -442,12 +442,12 @@ behavior, which can improve the performance of class loading from the shared cla
 :    - With the 31-bit VM on all versions of z/OS, the shared cache is memory mapped within the 0-2 GB address range. The maximum size of the persistent shared classes cache is limited by the system limit `MAXMMAPAREA`. For more information about the suggested value for the `MAXMMAPAREA` limit, see [Setting resource limits (z/OS)](https://www.eclipse.org/openj9/docs/configuring/#setting-resource-limits-zos).
      - With the 64-bit VM, the shared cache is mapped as follows:
          - On z/OS version 2.3 and earlier, the persistent shared cache is memory mapped within the 0-2 GB address range. The maximum size of the persistent shared classes cache is limited by the system limit `MAXMMAPAREA`.
-         - On z/OS version 2.4 and later, the persistent shared cache is mapped above the 2 GB address range. The maximum size of persistent shared classes cache is limited by the `MAXSHARE` value within the `SMFLIMxx` member of `SYS1.PARMLIB`. You can use the following machine configuration to create up to 1 GB shared classes cache mapped above the 2 GB address range:
+         - On z/OS version 2.4 and later, the persistent shared cache is mapped above the 2 GB address range. The maximum size of persistent shared classes cache is limited by the `MAXSHARE` value within the `SMFLIMxx` member of `SYS1.PARMLIB`. The default persistent shared cache size is 300MB, and the following machine configuration settings will allow a shared cache up to that size.
```

Comment:
Did someone try this configuration on a z/OS machine to confirm it is working as expected ? I am curious if the configuration allows a cache size < 300MB or cache size <=300MB.